EIPS/eip-parallel_tx_map.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Parallel Execution Maps
+description: Optionally include list of transactions without dependencies.
+author: Justin Florentine (@jflo) <justin@florentine.us>
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-05-22
+requires: [EIP-7685](eip-7685.md)
+---
+
+## Abstract
+
+With the advent of parallel transaction execution in some execution clients, it is now possible to share the safe execution path with other clients. When a client that supports parallel execution composes a block, and they know the set of independent transactions which can be executed out of order, they can share that set with all other clients which can then benefit from a parallel execution path.
+
+## Motivation
+
+On average, up to 70% of the transactions in a block are often independent of all the others. This large subset of the block may be executed in any order and still arrive at the same state root. The difficult part for clients is determining which transactions these are. 
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+This EIP introduces a new EIP-7685 request type `0x03` that clients MUST NOT treat as an error upon encountering.
+
+| Name                     | Value  | Comment                                                       |
+|--------------------------|--------|---------------------------------------------------------------|
+| `EXEC_HINT_REQUEST_TYPE` | `0x03` | The [EIP-7685](./eip-7685.md) type prefix for execution hints |
+
+The [EIP-7685](./eip-7685.md) encoding of an execution hint request is as follows. 
```

Comment:
Ok, this then ends up in the block header in the `requestsHash`. AFAIK currently the requests are not propagated over devp2p (neither should be since all of those can be re-calculated).

The client would then thus need to know what this data is in order to end up to the correct `requestsHash`. Since it is unclear how to write this txs list (plut it is optional) this would thus require that the data itself is somehow gotten to the client.

I like this idea, but I think we should setup some kind of "helper gossip" to exchange these kind of messages. As a hint: for this block you could also execute these txs in parallel. This would not be the way to go for "critical" blocks (tip-of-chain) but to re-execute the block this should/would be fine (?)

Reply:
Damn, ok most of the leverage for this idea depends on the requests being included in the block, not having to be added by the CL. Alternately, this could just be added to the block header as a new field, potentially styled after how requests work. 

Decoupling it from the block introduces timing complexity.

Reply:
This has been removed, and replaced with a new general purpose block metadata object.